specs/UserActivityHistory/UserActivityHistoryAPI.md

```diff
@@ -0,0 +1,243 @@
+interacting with.
+
+# API Pages
+
+## UserActivityHistory class
+
+This class provides static methods that enable you to query the user's activity history.
+This activity history is stored in a database managed by a local service, and these APIs
+call out to that service to retrieve data from the database.
+
+Here is an example usage of the class that will enable you to bring back the webpage for a
+Korean recipe that the user had previously interacted with within the last day:
+
+```c#
+UserActivityHistoryQuery query = new();
+query.Keywords = new string[] { "Korean", "recipe" };
+query.LatestStartTime = DateTime.Now.AddDays(-1);
+
+IList<UserActivityHistoryItem> results = await UserActivityHistory.SearchAsync(
+    new UserActivityHistoryQuery[] { query },
+    UserActivityHistoryOrderBy.DwellTime,
+    maxResults: 1);
+
+UserActivityHistoryItem item = results.FirstOrDefault();
+
+if (item != null)
+{
+    // Now we can use item.ActivationUri to bring back the webpage in the state in which the user
+    // was last viewing it.
+}
+```
+
+## UserActivityHistory.Search method
+
+This method synchronously queries the user's activity history and returns a list of items matching
+the criteria specified in the `queries` parameter. The results are ordered in descending order
+according by the `orderBy` parameter: either by the most recent start times, the most recent
+end times, or the longest time spent on the activity.
+
+Each parameter in `queries` is ORed together in the resulting database query, whereas the contents
+of a single object in the `queries` array are ANDed together.  For example, if you provided two
+queries, each of which contained the keywords "tax", one of which had a content type of
+"application/pdf" and the other had a content type of "image/*", the resulting database query
+would be something along these lines:
+
+```sql
+SELECT * FROM UserActivityHistory WHERE
+    (CONTAINS (DisplayText, '"tax"')) AND
+    ((CONTAINS (ContentType, '"application/pdf"')) OR
+     (CONTAINS (ContentType, '"image/*"')));
+```
+
+## UserActivityHistory.SearchAsync method
+
+This method is an asynchronous version of the `Search` method.
+
+## UserActivityHistory.GetAppsWithUserActivity method
+
+This method synchronously retrieves a list of all the app names with data in the user's activity
+history database. You can use this, for example, to show the user the list of apps that are being
+queried against, so the user can understand why an app that is not recording user activity is not
+showing up in the results.
+
+## UserActivityHistory.GetAppsWithUserActivityAsync method
+
+This method is an asynchronous version of the `GetAppsWithUserActivity` method.
+
+## UserActivityHistoryItem class
+
+This class represents a single item in the user's activity history. It contains properties that
+describe in what app the activity occurred, what the nature of the activity was, the URI of the
```

Comment:
can you clarify what you mean by "nature of the activity"?

Reply:
Updated to explicitly call out what's in a user activity object.